bindep.txt

```diff
@@ -16,6 +16,8 @@ texlive-tex-gyre [platform:rpm]
 texlive-pict2e [platform:rpm]
 texlive-fontawesome5 [platform:rpm]
 texlive-xcolor [platform:rpm]
+texlive-unicode-math [platform:rpm]
+texlive-xits [platform:rpm]
```

Comment:
Actually, [it seems all are "subpackages"](https://packages.fedoraproject.org/pkgs/texlive/) of ``texlive`` which we require earlier in this bindep.txt for Fedora, see also [here](https://src.fedoraproject.org/rpms/texlive/blob/rawhide/f/sources), so I am not sure we really need this and/or any of the other lines.

As per Ubuntu, unicode-math is part of texlive-latex-recommended, and XITS Math of texlive-fonts-extras which we both require already.